### ISSUE TITLE
Database Navigator can't drop aliases ("procedures") in H2

### ISSUE THREAD

**Issue by lukaseder** [dbeaver/dbeaver#12971]
#### System information: 
- **Operating system (distribution) and version:** Microsoft Windows [Version 10.0.19042.1052]
- **DBeaver version:** Version 21.1.1.202106210824

#### Connection specification:
- **Database name and version:** H2 1.4.200
- **Driver name:** `org.h2.Driver`
- **Do you use tunnels or proxies (SSH, SOCKS, etc)?** No

#### Describe the problem you're observing:

I can't use the Database Navigator to drop aliases in H2 ("procedures")

#### Steps to reproduce, if exist:

Using the H2 database, create an alias (their name for stored function):

```java
CREATE ALIAS MY_SQRT FOR "java.lang.Math.sqrt";
SELECT MY_SQRT(4);
```

Refresh the Database Navigator view to see it appear under "Procedures":

![image](https://user-images.githubusercontent.com/734593/123253252-8fcf5780-d4ed-11eb-88d6-2852dff4f21a.png)

Now click on delete from the context menu, and you can see that a `DROP FUNCTION` command is created:

![image](https://user-images.githubusercontent.com/734593/123253308-a2499100-d4ed-11eb-9024-bf11df5e4553.png)

Unfortunately, that's invalid in H2:

![image](https://user-images.githubusercontent.com/734593/123253338-ab3a6280-d4ed-11eb-9ae7-c26352cbe166.png)

It should be `DROP ALIAS`: https://h2database.com/html/commands.html#drop_alias

Note you might run into https://github.com/dbeaver/dbeaver/issues/11428 if you're running this example on a JRE instead of a JDK

**Comment by Matvey16** [comment]
Thank you for the report

**Comment by serge-rider** [comment]
We need to add aliases support in H2

**Comment by LonwoLonwo** [comment]
Drop statement fixed in DBeaver 21.2.1

![133213228-220580f5-402a-4e98-8888-b20542cf47f4](https://user-images.githubusercontent.com/45152336/133274505-ac28cb45-177b-4b4e-a5c9-018a01309b71.png)

Also, the source tab now shows the proper alias script and I added the ability to change alias description via UI.

![133213227-d29bd84d-109b-4d2c-8ef3-b225510e4f06](https://user-images.githubusercontent.com/45152336/133275786-024a797f-b672-4d02-bff3-bed6484e4964.png)

**Comment by uslss** [comment]
verified
